pages/common/gnmic.md

```diff
@@ -0,0 +1,24 @@
+# gnmic
+
+> A gNMI CLI client that provides full support for Capabilities, Get, Set and Subscribe RPCs with collector capabilities.
+> More information: <https://gnmic.kmrd.dev>.
+
+- Request capabilities (gnmic capabilities):
+
+`gnmic --address {{ip:port}} --insecure capabilities`
+
+- Username and password can be given to any command:
```

Comment:
Command examples typically describe what a command is doing and how it's used, can we indicate 

```suggestion
- Provide a username and password to fetch device capabilities:
```